test-apps/tutorial-servo-can-control/src/TutorialServo.cpp

```diff
@@ -0,0 +1,35 @@
+#pragma once
+#include "mbed.h"
+#include "TutorialServo.h"
+
+TutorialServo::TutorialServo(PinName servoPin, float servoRangeInDegrees, float
+minPulsewidthInMs, float maxPulsewidthInMs):
+m_servoPwmOut(servoPin),
+m_servoRangeInDegrees(servoRangeInDegrees),
+m_minPulsewidthInMs(minPulsewidthInMs),
+m_maxPulsewidthInMs(maxPulsewidthInMs){
+    m_servoPwmOut.period_ms(20);
+}
+
+void TutorialServo::setPositionInDegrees( const float degrees){
+    if (degrees <= m_servoRangeInDegrees && degrees >= 0)
+    {
+        m_servoPwmOut.pulsewidth((degrees/m_servoRangeInDegrees)*(m_maxPulsewidthInMs-m_minPulsewidthInMs));
```

Comment:
what if `m_minPulsewidthInMs` is nonzero? you'll need the constant offset so that 0 degrees maps to `m_minPulsewidthInMs`